benches/benchmark_portscan.rs

```diff
@@ -44,7 +46,8 @@ fn portscan_udp() {
         vec![],
         true,
     );
-    // Perform the actual scan or logic here if needed
+
+    let _ = block_on(scanner.run());
```

Comment:
Why `let`? 🤔

Reply:
Ahh I meant like in Python you could just call the function and not do `let`, I was wondering if Rust is the same 🤔

Reply:
TBH it's probably too many years of working with Python that made me use the `let`: if you don't assign things to a variable, sometimes you end up with random things going to stdout as they've nowhere else to go.

Pretty sure the Rust compiler is smarter than that though.